metadata.json

```diff
@@ -26,7 +26,7 @@
     },
     {
       "name": "katello/certs",
-      "version_requirement": ">= 6.1.1 < 9.0.0"
+      "version_requirement": ">= 8.0.0 < 9.0.0"
```

Comment:
Technically this should be the next version since it's only in git master. That makes me think we should increase the version in puppet-certs master already, even if we don't release. Looking at the changes there, https://github.com/theforeman/puppet-certs/pull/289#issuecomment-656667661 makes me think we should already bump to 9.0.0. Doing that now also has the benefit that we don't have to do it right before the release.

Reply:
Remind me that process? Is it the same as the release process or is it just literally bumping the version in metadata.json?

Reply:
Right now I'm suggesting to only bump the version in metadata.json. Then we also need to update the consuming modules (katello and foreman_proxy_content) to allow it.

Thinking more about it, I suspect we'll need more breaking changes before we do a real release so let's just go with 9.0.0.

Reply:
Bump: https://github.com/theforeman/puppet-certs/pull/292

Reply:
Bump for FPC -- https://github.com/theforeman/puppet-foreman_proxy_content/pull/273